File: backend/Watcher.Common/MoveToFrontend/organization.ts

 import { Instance } from './instance';
import { Notification } from './notification';

    export interface Organization  { 
        id: number;
        name: string;
        description: string;
        email: string;
        webSite: string;
        contactNumber: string;
        isActive: boolean;
        themeId: number;
        chatId: number;
        createdByUserId: number;
        usersId: number[];
        instances: Instance[];
        notifications: Notification[];
    }

File: backend/Watcher.Common/Dtos/Organization.cs
using System.Collections.Generic;
using Watcher.Common.MoveToFrontend;

namespace Watcher.Common.Dtos
{
    [ExportClassToTypescript]
    public class Organization : Entity<int>
    {
        public override int Id { get; set; }
        public string Name { get; set; }

        public int ThemeId { get; set; }
        public Theme Theme { get; set; }

        public Chat Chat { get; set; }

        public IList<UserOrganization> UserOrganizations { get; set; }

        public IList<Instance> Instances { get; set; }

        public IList<Notification> Notifications { get; set; }
    }
}
